1 porting

samueledonato opened the pull request.

samueledonato committed:
Issue #1: Completed porting to Lf 7.1 CE/DXP. Tested in:
-Liferay DXP 7.1 sp1
-Liferay CE 7.1 GA2
-Liferay CE 7.1 GA3

samueledonato committed:
#Issue #1: Porting Liferay 7.1 and bug fix of cookie "OK" button

samueledonato committed:
Issue 1#: Source Formatting

mmariuzzo reviewed (changes requested):
- There are sources not part of the project.
- No use of 'internal' package
- PortletKeys have to be shortened
- Copyright (module is distributed as LGPL) 
- ensure files use Unix EOL (no CR-LF)

samueledonato committed:
Correct the following errors:
 - There are sources not part of the project.
 - Copyright (module is distributed as LGPL)
 - ensure files use Unix EOL (no CR-LF)

samueledonato committed:
Issue #1: Modified Copyright and delete module frontend-js-spa-transitions-web
